src/app/public/modules/tabs/tab-button.component.html

```diff
@@ -10,6 +10,10 @@
       'sky-tab-btn-closeable': allowClose
     }"
     [tabindex]="disabled ? '-1' : '0'"
+    [routerLink]="routerLink"
+    [queryParams]="queryParams"
```

Comment:
Code style > ordering of properties is a little off here.

Reply:
I had the same thought, but I'm leaning toward keeping the properties in this order to keep the like-minded directives together and avoid confusion (e.g., `routerLink` is the directive, and the next few inputs supplement that). I'll change it to alphabetical ordering if you feel strongly, I just didn't want to confuse future engineers if they saw `preserveFragment` out of context.

Reply:
Hmm... valid concern. I faced this same issue with some new directives. Instead of adding exceptions to the code-style rule, I wondered if it was shining a light on a bigger problem - its not clear that a directive's properties belong to together. Instead of changing the alphabetical rule, I opted to prefix the directives' inputs so they were always together (and could still be alphabetized). What would you think about `routerLink` and `routerQueryParams`?

Reply:
Good idea, but all of these attributes belong to Angular. :-/

Reply:
Oh! Duh...
I dunno man, it feels bad to add exceptions to an already very long list of rules. We either need to follow them to the letter, or change the rules so they apply to this scenario. Otherwise, we'll start taking them less seriously. For the amount of time we spend in reviews going back and forth on code style, it seems like we already have a culture of beings strict with our style.